examples/with-iron-session/lib/hooks/index.js

```diff
@@ -0,0 +1 @@
+export { useUser } from './useUser'
```

Comment:
Do we need this file? for the same of simplicity the import can go to `hooks/useUser` directly.

Reply:
Removed and went for hooks/useUser in e53b730ee3f0e078285a5dc99afe8a4f6678e8d9